docs/examples-and-tutorials/videos/index.md

```diff
@@ -0,0 +1,41 @@
+---
+id: videos
+title: Videos
+sidebar_position: 5
+---
+
+# 🎥️ Videos
+
+## Office Hours
+
+- [Developer Self-Service](https://www.youtube.com/watch?v=74qMbqWBxms&list=PL2Vye-us8_x8-IErakx_KggCC7EPWJuby&index=2)
+- [Troubleshooting Kubernetes in Microsoft Teams](https://youtu.be/HaHaa9ju8Wg?feature=shared)
+- [GitOps Practices with Flux](https://www.youtube.com/watch?v=WxkDWLqVghw&list=PL2Vye-us8_x8-IErakx_KggCC7EPWJuby)
+- [Developer Productivity w/ Oketeo](https://www.youtube.com/live/506Tc1Grj_o?feature=shared)
+- [Botkube goes GitOps](https://www.youtube.com/watch?v=WxkDWLqVghw)
+- [Kubernetes Multi Cluster Management](https://www.youtube.com/live/ZeaUfboNKfc?feature=shared)
+- [5 Essential Kubernetes Tasks to Automate](https://www.youtube.com/live/CzlSHMM_Ypc?feature=shared)
+- [ArgoCD Workflows](https://youtu.be/1i59PN-65SY?feature=shared)
+- [Zapier for Kubernetes](https://www.youtube.com/live/VJzSRTQpHhs?feature=shared)
+
+## Community calls
+
+- [Open Source Friday- ChatOps for Kubernetes](https://www.youtube.com/live/tdyPuMA2mww?feature=shared)
+- [CNCF Webinar- Unlocking the Power of Kubernetes Collaborative Troubleshooting](https://youtu.be/6VTEOOfIbIk?feature=shared)
+- [Tailwarden- ChatOps for Kubernetes](https://youtu.be/UOURPb03mxU?feature=shared)
+- [Kubefirst- Monitor & Act on Your Kubernetes Cluster](https://www.youtube.com/live/RLmU472lync?feature=shared)
+- [CNCF Islamabad- Unlocking K8s Troubleshooting Best Practices](https://www.youtube.com/live/Ry_UijS1ZNk?feature=shared)
```

Comment:
Fix spacing (should be space before word and dash)
[Open Source Friday - ChatOps for Kubernetes]